### PR TITLE
Feature/grunt 1.2

### LINKED ISSUE #70: Can't install with Grunt 1.2.1
Hi! I'm using Grunt 1.2.1, but I can't install your package, because you hard-coded to use Grunt `~1.1` only. It would be great if you will not put such strict limitations. Thank you!

### PR DESCRIPTION
**Pull request by rvalitov**
Allow to use Grunt 1.1 and later. Fixes #70

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,14 +32,14 @@
   "devDependencies": {
     "eslint": "7.1.0",
     "eslint-config-paazmaya": "7.0.0",
-    "grunt": "1.1.0",
+    "grunt": "^1.2.1",
```
Comment (paazmaya) [inline package.json:35]: Please lock this to the given version, meaning without the prefix

Comment (rvalitov) [reply package.json:35]: I suppose it should be `^1.1` because 1.1 is the minimal version that your package require?

Comment (rvalitov) [reply package.json]: Done

**Comment by paazmaya** [comment]
Thank you very much for your contribution

**Pull request merged**